src/platform/datapath_winuser.c

```diff
@@ -3360,6 +3366,7 @@ CxPlatDataPathUdpRecvComplete(
 BOOLEAN
 CxPlatDataPathStartReceive(
     _In_ CXPLAT_SOCKET_PROC* SocketProc,
+    _Out_opt_ ULONG* IoResult,
     _Out_opt_ uint16_t* InlineBytesTransferred
     )
```

Comment:
We need to return the raw IO result here because we don't want to have to support three different IO status types: Win32 error, Winsock error, and QUIC_STATUS. It would be nice to refactor this further.